### PR TITLE
Update for .NET Core RC2.

### PR DESCRIPTION
**Pull request by jeremymeng**
The dependency System.Console 4.0.0-rc2-24027 supports netstandard1.3, thus the same netstandard version is chosen here.

This is for #2.

I mostly copied changes from commits of other sinks. The project builds fine in VS after restoring packages.  How do I test the powshell build script?

### CODE REVIEW

**Comment by merbla** [comment]
Hey @jeremymeng,

Thanks for the PR.

Two items re: the build
- First, you will need to update the build script to [this one](https://github.com/serilog/serilog-sinks-console/blob/dev/Build.ps1).  It seems to match so its just a case of running the `ps1` file.
- Secondly, the location for the `install.ps1` moved just after RC2.  It now lives here. See [Console Sink AppVeyor](https://github.com/serilog/serilog-sinks-console/blob/dev/appveyor.yml#L6).

**Comment by jeremymeng** [comment]
@merbla I was told by the team that the link to dotnet_install.ps1 is the ever-changing tip of the development branch.  If we want a stable one we should use a branched/released version, for example [https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0**-preview1**/scripts/obtain/install.ps1](https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0-preview1/scripts/obtain/install.ps1).  What do you think?

The `Build.ps1` failed at the test.  Of course I don't have the `test/Serilog.Tests` folder.  How do you normally set it up? Clone the `Serilog` repo then copy it over?

**Comment by merbla** [comment]
Thats interesting to know re: the build script. ping @nblumhardt @khellang 

Sorry as for `Build.ps1` you are correct, there are not any tests.  When the Serilog repo was split the old core tests were split to the respective sinks and this one did not have any.  Happy for you add a check to test the existence of the test folder or remove for the short term.

**Comment by jeremymeng** [comment]
Ah, got it.  I think it's better to follow the `Console` sink which doesn't have tests either.  PR updated.

File: .#Build.ps1 [hunk 1]
```diff
@@ -0,0 +1 @@
+yumeng@YU-Z230.660:1465299344
```
Comment (nblumhardt) [inline .#Build.ps1]: This one looks like a temporary file that should probably be ignored.

Comment (jeremymeng) [inline .#Build.ps1]: Done.  Thanks!

**Pull request merged**